tedana/tests/test_stats.py

```diff
@@ -1,13 +1,80 @@
 """
-Tests for tedana.stats.get_coeffs
+Tests for the tedana stats module
 """
-
 import numpy as np
 import pytest
+import random
 
+from tedana.stats import computefeats2
 from tedana.stats import get_coeffs
+from tedana.stats import getfbounds
+
+
+"""
```

Comment:
I'm a little confused having these block comments outside of the functions. Is there a strong reason to have them, here ?